CGAL_ImageIO/src/CGAL_ImageIO/CMakeLists.txt

```diff
@@ -42,36 +42,40 @@ if(NOT CGAL_HEADER_ONLY)
 
     use_essential_libs()
 
-    include_directories( SYSTEM ${CGAL_3RD_PARTY_INCLUDE_DIRS} ${CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS} .)
-
-    link_directories    ( ${CGAL_LIBRARIES_DIR} ${CGAL_3RD_PARTY_LIBRARIES_DIRS} )
   endif(OPENGL_FOUND)
 
-else (NOT CGAL_HEADER_ONLY)
-  cache_set(CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR})
+if (CGAL_HEADER_ONLY)
+  cache_set(CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS ${CGAL_ImageIO_3RD_PARTY_INCLUDE_DIRS} ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
```

Comment:
I think there is a bug, here. Should not the `if(CGAL_HEADER_ONLY)` be a `if(NOT CGAL_HEADER_ONLY)`?